roles/openshift_logging/templates/elasticsearch.yml.j2

```diff
@@ -7,8 +7,7 @@ script:
 
 index:
   number_of_shards: 1
-  number_of_replicas: 0
-  auto_expand_replicas: 0-2
+  number_of_replicas: 1
```

Comment:
Can I make the argument that this is something we can specify as part of the install? 
Or if a customer states they want an ES cluster size of `2` or more we would then change this value?

Or or - if not specified we would change this value based on the intended cluster size (0 - 2) but then give users the ability to overwrite it if they so desire.